.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,34 @@
 ## Describe the PR
-<!-- A clear and concise description of the bug the PR fixes or the feature the PR introduces. We may use this for the changelog and/or documentation. -->
+<!-- 
+A clear and concise description of the bug the PR fixes or the feature the PR introduces. 
+We may use this for the changelog and/or documentation. 
+-->
 
-## Related issues
+## Breaking changes
+<!-- If PR creates known breaking changes, please list them: -->
+
+## Related issues/ideas
 <!-- PR relates to issues in the `kirby` repo or ideas on `feedback.getkirby.com`: -->
 
 - Fixes #
 
 ## Ready?
-<!-- If you feel like you can help to check off the following tasks, that'd be great. If not, don't worry - we will take care of it. -->
+<!-- 
+If you feel like you can help to check off the following tasks, 
+that'd be great. If not, don't worry - we will take care of it. 
+-->
+
+- [ ] Unit tests for fixed bug/feature
+- [ ] In-code documentation (wherever needed)
+- [ ] CI checks pass
 
-- [ ] Added unit tests for fixed bug/feature
-- [ ] Added in-code documentation (if needed)
-- [ ] CI passes (runs automatically when the PR is created or run `composer ci` locally)
-  Running locally requires PHPUnit, PHP-CS-Fixer, Psalm, PHPCPD and PHPMD.
+<!-- 
+CI runs automatically when the PR is created or run `composer ci` locally.
+Running locally requires PHPUnit, PHP-CS-Fixer, Psalm, PHPCPD and PHPMD.
+-->
 
-<!-- We will take care of the following TODO when reviewing the PR. -->
+## When merging
+<!-- We will take care of the following TODO when reviewing and merging the PR -->
 
-- [ ] Checked whether the PR needs documentation, if needed added to the [release docs checklist](https://github.com/getkirby/getkirby.com/pulls)
+- [ ] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
+- [ ] Add breaking changes to [release notes draft](https://github.com/getkirby/kirby/releases)
```

Comment:
Is that our current workflow? As far as I can tell, @bastianallgeier writes the release notes on release by going through the PRs in the milestone.

Using a release notes draft would be great as well, but we would need to do it consistently. In this case not only the breaking changes should be added, but the whole PR should be listed with its enhancements, fixes and features.

Reply:
It's not but I thought it should be. I recalled that with 3.5 we indeed had some breaking changes we thought of but then forgot to put into the release notes. That's why I would like to make breaking changes more explicit in PRs and directly gather them somewhere, e.g. a release draft.

I'd agree that directly drafting the rest of the release notes with each PR would be good indeed. One can still rearrange and rewrite some parts at release, but it makes it easier to already have all the changes mentioned.

Reply:
Yep, it would likely also reduce the work on release for Bastian.

I think we need his opinion on the way forward. :)

Reply:
This would be so great! I go through the commits and PRs manually, but it's often super difficult to extract everything and don't miss important stuff. I'm only critical of a release draft. While it would be the best place to store such information, those drafts can be published accidentally, but just clicking the wrong button. When we keep such a draft around for weeks, I'm pretty sure we will unintentionally publish such drafts way too often.

Reply:
@bastianallgeier where else would work well?

Reply:
Would Notion work for us? Or maybe an additional issue just for the release notes?

Reply:
I like Notion, we can add a section to our Plans page for the release

Reply:
Changed the language to refer to Notion